lib/addons/aws-privateca-issuer/index.ts

```diff
@@ -0,0 +1,95 @@
+// lib/kubevious_addon.ts
+import { Construct } from 'constructs';
+import { ManagedPolicy } from "aws-cdk-lib/aws-iam";
+import merge from "ts-deepmerge";
+import { HelmAddOn, HelmAddOnUserProps, HelmAddOnProps } from "../helm-addon";
+import { dependable } from '../../utils';

```

Comment:
Let's group all things that you import from utils in a single import. 
`import { dependable, setPath etc. } from "../../utils";`

Reply:
done